### ISSUE TITLE
doubt about multi machine training for wav2letter

### ISSUE THREAD

**Issue by bolt163** [facebookresearch/wav2letter#60]
As i mentioned in the last issue[https://github.com/facebookresearch/wav2letter/issues/57],the training time still need 80+days if i still training on  GPU(Nvidia Tesla K40M,with 12G memory), using  mpi with 3GPU(for each epoch it still need 2days more), 

while there is a small crowd of  server,so what I want to do is using multi machine to train different training dataset , and  merge the model generated by each machine  together for finally using, this may be the available way for me for accelerating the training process 

If I train the total training data set on 4 machine(for each,  mpi training with  3 gpu),there will be 4 models  namely mode1.bin/model2.bin/model3.bin/model4.bin  generated, for totally using, how can I merge the 4 model together? is there any solution? thanks

**Comment by vineelpratap** [comment]
Hi,
 With MPI training, we synchronize with all process at each step and generate a single model. However, our GPUs were only dedicated to this training and were more powerful than K40Ms. Just to give an estimate, we used 4 processes in MPI and our runtime for each epoch was around 12hrs.

Regarding combining models, one could try averaging the outputs from each model but we noticed it didn't work well in our experiments.
